src/Symfony/Component/DependencyInjection/Tests/DefinitionTest.php

```diff
@@ -198,6 +198,7 @@ public function testSetIsDeprecated()
         $this->assertSame($def, $def->setDeprecated(true), '->setDeprecated() implements a fluent interface');
         $this->assertTrue($def->isDeprecated(), '->isDeprecated() returns true if the instance should not be used anymore.');
         $this->assertSame('The "deprecated_service" service is deprecated. You should stop using it, as it will soon be removed.', $def->getDeprecationMessage('deprecated_service'), '->getDeprecationMessage() should return a formatted message template');
+        $this->assertNull($def->setDeprecated(false)->getDeprecationMessage('deprecated_service'), '->getDeprecationMessage() returns null if the service is not deprecated');
```

Comment:
behavior change (see above) should be reverted, thus test case should be adjusted